lib/configuration.yaml

```diff
@@ -14979,6 +14979,13 @@ mainnet_dryrun_full: &mainnet_dryrun_full
     attribResrictEpoch: 2
     addrAttribSize: 128
     txAttribSize: 128
+  update: &mainnet_base_update
+    applicationName: cardano-sl
+    applicationVersion: 1
```

Comment:
I'd have to double check the app version here. What was it before?

Reply:
1